packages/frontend/lib/index.ts

```diff
@@ -33,8 +32,8 @@ export default class Nango {
         this.status = AuthorizationStatus.IDLE;
         this.publicKey = config.publicKey;
 
-        if (this.isCloud() && !config.publicKey) {
-            throw new Error('You should specify a Public Key when using Nango Cloud (cf. documentation).');
+        if (!config.publicKey) {
```

Comment:
My first instinct was that this would never happen. But it could if using js, right? I never realized that we should account for js type-ignorance when writing ts.